### PR TITLE
DOCS-3545 Adds Alerting with Conversion Rates Guide

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Adds Alerting with Conversion Rates guide.

### Motivation

DOCS-3545

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by jamiemilstein — CHANGES_REQUESTED** [review]
This looks good! Can we wait to merge it until ~tuesday of next week? the screenshots i took this on are in a new sidepanel that isn't going live until then

File: content/en/real_user_monitoring/guide/alerting-with-conversion-rates.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+The following example demonstrates how to create a monitor that alerts on the conversion rate of the `Product` step in the funnel.
+
+{{< img src="real_user_monitoring/guide/alerting-with-conversion-rates/second-step-in-funnel.png" alt="Create a monitor around the conversion rate of the second step in the funnel" style="width:100%;" >}}
+
+Create some steps from your views and actions. You can click on the bar graphs to see a side panel with analytics about user conversions and dropoffs. To add a subsequent view or action in the funnel, click **+** and select from frequent next steps. 
+
+## Export conversion rate graph
+
+The funnel displays the overall conversion and dropoff rates, number of converted or dropoff sessions, and percentage of converted or dropoff sessions.
```
Comment (apigirl) [inline content/en/real_user_monitoring/guide/alerting-with-conversion-rates.md:27]:
```suggestion
The funnel displays the overall conversion and dropoff rates, the number of converted or dropoff sessions, and the percentage of converted or dropoff sessions.  
```

**Review by apigirl — APPROVED** [review]
just a nit, this looks great!

**Comment by jamiemilstein** [comment]
these screenshots are going to be live by end of this week / early next. i dont see any harm in actually merging this now since the queries / content is all the same!

**Review by jamiemilstein — APPROVED** [review]

**Pull request merged**